elements/rh-audio-player/demo/rh-audio-player.js

```diff
@@ -0,0 +1,71 @@
+import '@rhds/elements/rh-audio-player/rh-audio-player.js';
+
+const root = document.querySelector('[data-demo="rh-audio-player"]')?.shadowRoot ?? document;
```

Comment:
we can just use document. root is a holdover from when we used to put demos in shadowroots